llvm/utils/repo/repo.cmake

```diff
@@ -24,10 +24,57 @@ else ()
     set (utils_dir /usr/share/repo)
 endif ()
 
-SET (CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0 -fno-exceptions -fno-rtti" CACHE STRING "Default C Flags Debug")
-SET (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0 -fno-exceptions -fno-rtti" CACHE STRING "Default CXX Flags Debug")
-SET (CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -O3 -fno-exceptions -fno-rtti" CACHE STRING "Default C Flags Release")
-SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -fno-exceptions -fno-rtti" CACHE STRING "Default CXX Flags Release")
+# The user may specify the libcxxabi_include by giving
```

Comment:
As elsewhere, it would be more useful to describe the function of the variable here.

Reply:
Fixed in the commit [6c4f10c](https://github.com/SNSystems/llvm-project-prepo/commit/6c4f10ca7cb881e1330eb14bea461620683715d0).